blurp-app/src/pages/testpage.jsx

```diff
@@ -4,6 +4,10 @@ import { SigmaContainer, useLoadGraph } from "@react-sigma/core";
 import "@react-sigma/core/lib/react-sigma.min.css";
 import { v4 as uuidv4 } from 'uuid';
 
+import SearchBar from "../components/searchbar.jsx";
```

Comment:
Maybe it might be better for us to create our own search bar but look into sigma React Search control component (part of react-sigma/core). 

Here is some documentation and the example shows the search bar that you can play around with 
https://sim51.github.io/react-sigma/docs/example/controls

Reply:
Let's use their dedicated search bar instead. It seems it's already mapped to the node/edge data as well. Putting in a ticket for change.